client/src/document/ingredients/browser-compatibility-table/accordion.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useContext, useMemo, useState } from "react";
+import type bcd from "mdn-browser-compat-data/types";
+
+type Id = [number, bcd.BrowserNames];
```

Comment:
What does this mean? 
/me still learning TS

Reply:
It's a tuple with 2 elements, where the first element's type is `number` and the second's type is `BrowserNames`. Possible assignments to that type are:
`[2, 'chrome']`, `[42, 'firefox']`,...
Impossible assignment are
`[4, 'not-a-browser']`, `[]`, `23`, `[2, 'chrome', 'wat]`